flexget/plugins/sites/limetorrents.py

```diff
@@ -1,18 +1,34 @@
 from __future__ import unicode_literals, division, absolute_import
+
+import re
 from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
 
 import logging
+from unicodedata import normalize
 
 from flexget import plugin
 from flexget.entry import Entry
 from flexget.event import event
 from flexget.utils.requests import RequestException
 from flexget.utils.soup import get_soup
-from flexget.utils.search import torrent_availability, clean_symbols
+from flexget.plugins.sites import torrent_availability
 from flexget.utils.tools import parse_filesize
 
 log = logging.getLogger('limetorrents')
 
+
+def clean_symbols(text):
```

Comment:
why is this in this plugin and not the main utils module?

Reply:
it's only used in this one plugin, I rather keep it that way